_includes/markdown/Design.md

```diff
@@ -171,12 +171,18 @@ If a management UI is impossible due to the nature of the project, credentials s
 <?php
 // Production API keys should ideally be defined in wp-config.php
 // This section should default to a development or noop key instead.
-if ( ! defined( 'CLIENT_MANDRILL_API_KEY' ) && ! ENV_DEVELOPMENT ) {
-	define( 'CLIENT_MANDRILL_API_KEY', '1234567890' );
+if ( ! defined( 'CLIENT_MANDRILL_API_KEY' ) {
+    if ( defined( 'WP_ENVIRONMENT_TYPE' ) && 'production' === wp_get_environment_type() ) {
+        define( 'CLIENT_MANDRILL_API_KEY', '1234567890' );
+    }
 }
 ```
 
-The `ENV_DEVELOPMENT` constant should always be set to `true` for local development and should be used whenever and wherever possible to prevent production-only functionality from triggering in a local environment.
+The `WP_ENVIRONMENT_TYPE` constant should always be set to `local` or `development` for local development and should be used whenever and wherever possible to prevent production-only functionality from triggering in a local environment.
```

Comment:
Recommend changing `local environment` to `non-production`.

Reply:
@gthayer This has been updated from `local` => `non-production`